invalid_vars_testfile.json

```diff
@@ -0,0 +1 @@
+{"BING": {"required": true, "secret": false, "type": "string"}, "BONG": {"required": true, "secret": false, "type": "string"}, "BONG": {"required": false, "secret": false, "type": "string"}}
```

Comment:
nit: newlines here and in the other file

Reply:
I removed the file because it was actually an accidental leftover. I created it dynamically and removed the line to delete it to see the content for manual testing.